dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/enrollment/AbstractEnrollmentService.java

```diff
@@ -725,7 +726,7 @@ public ImportSummary updateEnrollment( Enrollment enrollment, ImportOptions impo
         {
             ImportSummary importSummary = new ImportSummary( ImportStatus.ERROR ).incrementIgnored();
             importSummary.setConflicts( importConflicts );
-
+            importSummary.setReference(enrollment.getTrackedEntityInstance());
```

Comment:
```suggestion
            importSummary.setReference( enrollment.getTrackedEntityInstance() );
```

Reply:
This should in this case actually be the uid of the enrollment, not the tracked entity instance (enrollment.getEnrollment())